non-free.md

```diff
@@ -60,7 +60,7 @@
 ## Money, Budgeting and Management
 
   * [Anchor](http://theanchorapp.com/) - Anchor is an invoicing system that integrates with Stripe and Paypal. Includes features such as: reporting, dashboard and no client limit. ([Demo](http://theanchorapp.com/demo/admin-login)) `PHP`
-  * [FusionInvoice](https://www.fusioninvoice.com) `⊘ Proprietary` - FusionInvoice is an invoicing system for freelancers and small businesses. `PHP`
+  * [InvoicePlane](https://invoiceplane.com/) `⊘ Proprietary` - InvoicePlane is a self-hosted open source application for managing your quotes, invoices, clients and payments. `PHP`
```

Comment:
This line is inherently contradictory, I think. It has a proprietary license and is listed in non-free.md, but is actually an open source application? Huh?

Reply:
@evaryont The situation looks a bit muddled, and my background context is limited.

### PR context

I haven't used FusionInvoice or InvoicePlane.  
I came across FusionInvoice while consulting `non-free.md`.  
My PR was originally going to be for the removal of FusionInvoice.  
I returned to make the changes a week or two after noticing the closure.  
This new development with InvoicePlane had since occurred.

### The licenses themselves

| Product       | Version | License, Project Site | License, src | Note                   |
|:--------------|:--------|:----------------------|:-------------|:-----------------------|
| FusionInvoice | 2018-8  | [Main Site][1]        |              | EOL                    |
| InvoicePlane  | 1.0     | [Wiki][2]             |              | no current branch      |
| InvoicePlane  | 1.5.x   | Wiki redirects to 2.0 | [GitHub][3]  | master                 |
| InvoicePlane  | 2.0.x   | [Wiki][4]             | [GitHub][5]  | alpha, contents differ |


### Muddled aspects

1. FusionInvoice appears to have considered itself FOSS all along.
   - See the earlier of the two closure messages.
2. The EOL version of FusionInvoice intended to become the active edition of InvoicePlane (2.x) is still an alpha release.
   - [The lead dev notes][6] that stable and alpha are incompatible, substantially different products as it stands.
3. InvoicePlane's wiki and its github repo differ.
   - The wiki omits a license for CodeIgniter. The repo does not.
4. The stable (1.5) release doesn't have a normie-friendly way to see its license.
   - Read: 1.5.x's license can't be seen on the main site.

1 and 4 don't change the licensing status.  
But they might raise doubts about inclusion in the main list.(?)

3 calls into question what exactly's going on with the licensing altogether.

2 raises the question: __what software is the list recommending as of now?__

Ostensibly, the product once called FusionInvoice 2018 will be freely licensed, all petty details aside.

What's better for the list?

- explicitly identify the product as InvoicePlane 2 and move it to main list
- mark its licensing information as inconsistent, and do whatever


---

[1]: https://www.fusioninvoice.com/docs/2018/About-FusionInvoice/License
[2]: https://wiki.invoiceplane.com/en/1.0/general/license
[3]: https://github.com/InvoicePlane/InvoicePlane/blob/master/LICENSE.txt
[4]: https://wiki.invoiceplane.com/en/2.0/general/license
[5]: https://github.com/InvoicePlane/InvoicePlane/blob/v2.0.0/LICENSE.txt
[6]: https://community.invoiceplane.com/t/topic/6299